models/teamMember.js

```diff
@@ -26,6 +26,8 @@ module.exports = function (sequelize, DataTypes) {
     },
     timestamps: true,
   })
-  TeamMember.belongsTo(Team, { foreignKey: 'team_id' })
-  TeamMember.belongsTo(User, { foreignKey: 'user_id' })
+  TeamMember.belongsTo(Team, { foreignKey: 'team_id' });

```

Comment:
the build is still failing because you are using TeamMember.belongsTo in a wrong way you need to do it like this,
`
TeamMember.associate = (models) => {
    TeamMember.belongsTo(models.Team, { foreignKey: 'team_id'})
}
`
something like this, in your current implementation it could be possible that Team that you are importing is a empty object ... also remove the imports at the top they are not needed anymore

Reply:
for reference you can have a look at this file 
[here](https://github.com/worknenjoy/gitpay/blob/fc340f296b6a570020af909f69241264add237c0/models/task.js)